README.md

```diff
@@ -86,31 +86,31 @@ On the following examples, we'll be considering that we have a complete configur
 
 ### Read a secret in plain text (default)
 ```console
-$ vault get my_secret
+$ vault get my_secret value
 qwerty
 ```
 
 ### Read a secret in yaml format
 ```console
-$ vault get --yaml my_secret
+$ vault get --yaml my_secret value
 --- qwerty
 ...
 ```
 
 ### Write a secret
 ```console
-$ vault set my_other_secret supersecret
+$ vault set my_other_secret value=supersecret
 Done
 ```
 
 ### Read/write a secret outside the base path
 ```console
 $ export VAULT_CLI_BASE_PATH=myapp/
-$ vault set /global_secret sharedsecret
+$ vault set /global_secret value=sharedsecret
 Done
-$ vault get /global_secret
+$ vault get /global_secret value
 sharedsecret
-$ vault get global_secret
+$ vault get global_secret value
 Error: Secret not found
 $ unset VAULT_CLI_BASE_PATH
```

Comment:
Wow this part really deserves a rewriting, I had to read 3 times to understand what it does.

Reply:
(talking about the doc itself, not your change specifically)